recipes/mongo-tools/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "mongo-tools" %}
+{% set version = "3.6.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/mongodb/{{ name }}/archive/r{{ version }}.tar.gz
+    sha256: 826962d3660f9455c581d891c0ccbce7fe0a1da4ff5d56b03a8e10c56791018e
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }}  # [not win]
+    - git  # [not win]
+    - make  # [not win]
+    - sed  # [not win]
```

Comment:
Since you are skipping windows builds, you don't need the `# [not win]` tags here.

Reply:
good point!